test/expected/alter.out

```diff
@@ -536,8 +608,8 @@ SELECT * from _timescaledb_catalog.hypertable;
 SELECT * from _timescaledb_catalog.chunk;
  id | hypertable_id |      schema_name      |     table_name     
 ----+---------------+-----------------------+--------------------
- 20 |            11 | new_associated_schema | _hyper_11_20_chunk
- 21 |            11 | new_associated_schema | _hyper_11_21_chunk
+ 22 |            11 | new_associated_schema | _hyper_11_22_chunk
+ 23 |            11 | new_associated_schema | _hyper_11_23_chunk
 (2 rows)
```

Comment:
Could we somehow change this test so that earlier tests creating new chunks would not break this?

Reply:
Yeah, you can definitely change to `select id,hypertable_id,schema_name from _timescaledb_catalog.chunk`, since the test just needs to check that the schema was renamed.

Reply:
could I not include id in the select as that is also going to change if more tests are added before this point.
So, it would be `select hypertable_id,schema_name from _timescaledb_catalog.chunk`

Reply:
Sure. This will undoubtedly also change if/when we add more hypertables before this test, so we might still have to make this more robust in the future.

Reply:
You can harden this further if you want by 
1) on line 592 do ` SELECT hypertable_id as my_hypertable_id FROM create_hypertable('my_table','date', associated_schema_name => 'my_associated_schema') \gset`
2) change this line to `select schema_name, table_name from _timescaledb_catalog.chunk where hypertable_id = :my_hypertable_id;`